### ISSUE TITLE
Highway Builder doesnt cover floor lava[Bug] 

### ISSUE THREAD

**Issue by AryonLegacy** [MeteorDevelopment/meteor-client#1776]
### Describe the bug

Highway builder will fall into floor lava. 

### Steps to reproduce

```bash
1. Start highwaybuilder
2. Watch as it doesnt place a block underneath it and falls into the lava pool
3. Profit
```

### Meteor Version

v0.4.7-1362

### Minecraft Version

1.18

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported.
(I have checked every bug report on github)

- [X] This is a valid bug.
(I am able to reproduce this on the latest dev build)

**Comment by AryonLegacy** [comment]
Currently you have to activate Scaffold to safely afk tunnel, but its tedious to turn 2 mods and off everytime.

**Comment by Wide-Cat** [comment]
should be fixed 30437a09b9a0f89f6fc5421d4faa3b6db778dc02
